cmd/webhook/reportPortalWebhook.go

```diff
@@ -0,0 +1,151 @@
+package webhook
+
+import (
+	"fmt"
+	"github.com/redhat-appstudio/qe-tools/pkg/prow"
+	"github.com/redhat-appstudio/qe-tools/pkg/webhook"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/klog/v2"
+	"os"
+	"strings"
+)
+
+// AppStudio QE webhook configuration values will be used by default (if none are provided via env vars)
+const (
+	appstudioQESaltSecret       = "123456789"
+	appstudioQEWebhookTargetURL = "https://hook.pipelinesascode.com/EyFYTakxEgEy"
+
+	jobTypeParamName          = "job-type"
+	jobNameParamName          = "job-name"
+	repoOwnerParamName        = "repo-owner"
+	repoNameParamName         = "repo-name"
+	prNumberParamName         = "pr-number"
+	saltSecretParamName       = "salt-secret"
+	webhookTargetUrlParamName = "target-url"
+	jobSpecParamName          = "job-spec"
+
+	jobTypeEnv          = "JOB_TYPE"
+	jobNameEnv          = "JOB_NAME"
+	repoOwnerEnv        = "REPO_OWNER"
+	repoNameEnv         = "REPO_NAME"
+	prNumberEnv         = "PULL_NUMBER"
+	saltSecretEnv       = "WEBHOOK_SALT_SECRET"
+	webhookTargetUrlEnv = "WEBHOOK_TARGET_URL"
+	jobSpecEnv          = "JOB_SPEC"
+)
+
+var (
+	openshiftJobSpec *prow.OpenshiftJobSpec
+	jobType          string
+	jobName          string
+	repoOwner        string
+	repoName         string
+	prNumber         string
+	saltSecret       string
+	webhookTargetUrl string
+	jobSpec          string
+)
+
+var reportPortalWebhookCmd = &cobra.Command{
+	Use: "report-portal",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if jobType == "" {
+			return fmt.Errorf("parameter '%s' and env var '%s' is empty", jobTypeParamName, jobTypeEnv)
+
+		}
+		if jobName == "" {
+			return fmt.Errorf("parameter '%s' and env var '%s' is empty", jobNameParamName, jobNameEnv)
+		}
+		if repoOwner == "" {
+			return fmt.Errorf("parameter '%s' and env var '%s' is empty", repoOwnerParamName, repoOwnerEnv)
+		}
+		if repoName == "" {
+			return fmt.Errorf("parameter '%s' and env var '%s' is empty", repoNameParamName, repoNameEnv)
+		}
+		if prNumber == "" {
+			return fmt.Errorf("parameter '%s' and env var '%s' is empty", prNumberParamName, prNumberEnv)
+		}
+		if jobSpec == "" {
+			return fmt.Errorf("parameter '%s' and env var '%s' is empty", jobSpecParamName, jobSpecEnv)
+		}
```

Comment:
wdyt about shortening this with `for` loop similarly like [here](https://github.com/redhat-appstudio/qe-tools/blob/5a211eb408f9dc388c2c4dff6119fb90022396f8/cmd/coffeebreak/coffeeBreak.go#L25-L31)